task3/frontpage.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="frontpage.css"> 
+<head>
+   <!-- <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">-->
+    <title>NTW</title>
+</head>
+<body  >
+    <header class="blok "  >
+        
+        <div class="d1 box0"></div>
+        
+        <div class="d2 box1 ">
+        
+            <p class="avtor">NTW Сальдюн Андрей 2 курс 7 группа  asaldyun@gamil.com  v.0.0</p>
+            
+            <form action="sing in"> 
```

Comment:
I said that the form should relates to the action, but one clarification: the action related to user input.
so one button is not the form
one textfield + button is form (or single text field in some cases)
So another words form should contain user input.

'+ action attribute define the URL that form should use to send data that not just description
To stub and prevent form from sending data you can use
`action="javascript: alert('Action happened')"`